src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -169,4 +198,100 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
             throw new RuntimeException("Error getting KitRequests ", e);
         }
     }
-}
+
+    public static void addOtherUnorderedKitsToList(Map<KitRequestSettings, ArrayList<KitRequest>> kitsToOrder) {
+        for (KitRequestSettings kitRequestSettings : kitsToOrder.keySet()) {
+            String query = "SELECT * FROM dev_dsm_db.ddp_kit_request request LEFT JOIN dev_dsm_db.ddp_kit kit on kit.dsm_kit_request_id = request.dsm_kit_request_id " +
+                    "LEFT JOIN ddp_kit_request_settings settings on settings.ddp_instance_id = request.ddp_instance_id " +
+                    "LEFT JOIN ddp_instance realm on (realm.ddp_instance_id = settings.ddp_instance_id) " +
+                    "where request.ddp_instance_id = ? and kit.needs_approval = true and kit.authorization = true and request.external_order_status is null";
```

Comment:
move query up please

Reply:
you need to remove the db here! (dev_dsm_db)

Reply:
Great catch! thanks